mappings/net/minecraft/util/Util.mapping

```diff
@@ -100,9 +112,21 @@ CLASS net/minecraft/unmapped/C_lvarvugc net/minecraft/util/Util
 	METHOD m_rjgutuoq (Ljava/nio/file/spi/FileSystemProvider;)Z
 		ARG 0 fileSystemProvider
 	METHOD m_rmfqlirq getOperatingSystem ()Lnet/minecraft/unmapped/C_lvarvugc$C_fqjbnqwz;
+	METHOD m_royqyqxg backupAndReplace (Ljava/nio/file/Path;Ljava/nio/file/Path;Ljava/nio/file/Path;Z)V
+		ARG 0 current
+		ARG 1 newPath
+		ARG 2 backup
+		ARG 3 noRestoreOnFail
```

Comment:
```suggestion
		ARG 3 preventRestoreOnFail
```
to comply with
> Boolean field and variable names should always be adjective phrases or present tense verb phrases

from [CONVENTIONS:General](https://github.com/QuiltMC/quilt-mappings/blob/1.18.2/CONVENTIONS.md#general).

Also consider
```suggestion
		ARG 3 discardOnFail
```
to avoid the double negative?